### PR TITLE
Update Finnish strings for 1.8

### PR DESCRIPTION
**Pull request by valtlai**

### CODE REVIEW

File: src/nls/fi/strings.js [hunk 1]
```diff
@@ -456,15 +459,15 @@ define({
     "PROJECT_SETTINGS_TITLE"               : "Projektin asetukset kohteelle: {0}",
     "PROJECT_SETTING_BASE_URL"             : "Esikatselun perus-URL",
     "PROJECT_SETTING_BASE_URL_HINT"        : "Käytä paikall. palvelinta antamalla URL, kuten http://localhost:8000/",
-    "BASEURL_ERROR_INVALID_PROTOCOL"       : "Esikatselu ei tue {0}-protokollaa. Käytä joko http: tai https: .",
+    "BASEURL_ERROR_INVALID_PROTOCOL"       : "Esikatselu ei tue {0}-protokollaa. Käytä joko http: tai https: -mallia.",
```
Comment (petetnt) [inline src/nls/fi/strings.js]: Maybe `muotoa`?

File: src/nls/fi/strings.js [hunk 2]
```diff
@@ -494,31 +497,32 @@ define({
...
-    "CANCELING_HUNG"                       : "Asennuksen peruutus kestää pitkään. On saattanut tapahtua sisäinen virhe.",
+    "CANCELING_HUNG"                       : "Asennuksen peruuttaminen kestää pitkään. On saattanut tapahtua sisäinen virhe.",
     "INSTALL_CANCELED"                     : "Asennus on peruutettu.",
     "VIEW_COMPLETE_DESCRIPTION"            : "Näytä täysi kuvaus",
     "VIEW_TRUNCATED_DESCRIPTION"           : "Näytä katkaistu kuvaus",
     // These must match the error codes in ExtensionsDomain.Errors.* :
     "INVALID_ZIP_FILE"                     : "Ladattu sisältö ei ole kelvollinen zip-tiedosto.",
-    "INVALID_PACKAGE_JSON"                 : "Tiedosto package.json on virheellinen. (virhe: {0}).",
-    "MISSING_PACKAGE_NAME"                 : "Tiedostolle package.json ei ole määritelty paketin nimeä.",
+	"MISSING_PACKAGE_JSON"                 : "Paketissa ei ole tiedostoa package.json.",
```
Comment (petetnt) [inline src/nls/fi/strings.js]: Tabs -> spaces

File: src/nls/fi/strings.js [hunk 3]
```diff
@@ -625,12 +629,12 @@ define({
...
+    "HEALTH_DATA_PREVIEW_INTRO"                 : "<p>Jotta voit parantaa {APP_NAME}ia, lähetämme säännöllisesti Adobelle rajattuja, <strong>nimettömiä</strong> tilastotietoja siitä, kuinka käytät tuotetta. Nämä tiedot auttavat priorisoimaan ominaisuuksia, löytämään bugeja ja huomaamaan käytettävyysongelmia. <a href='https://github.com/adobe/brackets/wiki/Health-Data'>Lue lisää {APP_NAME}in terveydentilaraportista</a> ja siitä, kuinka se hyödyttää {APP_NAME}-yhteisöä samalla, kun suojelemme yksityisyyttäsi.</p><p>Alla on esikatselu tiedoista, jotka lähetetään seuraavassa terveydentilaraportissasi <em>jos</em> se on otettu käyttöön.</p>",
```
Comment (petetnt) [inline src/nls/fi/strings.js]: IMHO should be something like `Jotta voimme parantaa`

File: src/nls/fi/strings.js [hunk 4]
```diff
@@ -681,11 +690,11 @@ define({
...
     "DESCRIPTION_CODE_FOLDING_ENABLED"               : "true: ota koodin laskostus käyttöön",
     "DESCRIPTION_CODE_FOLDING_HIDE_UNTIL_MOUSEOVER"  : "true: näytä osion pienennyksen merkit vain, kun osoitin siirretään vasemman marginaalin päälle",
-    "DESCRIPTION_CODE_FOLDING_MAX_FOLD_LEVEL"        : "Rajoittaa sitä, kuinka monta tasoa syvälle Pienennä kaikki \u2011valinta vaikuttaa",
+    "DESCRIPTION_CODE_FOLDING_MAX_FOLD_LEVEL"        : "Rajoittaa sitä, kuinka monta tasoa syvälle Pienennä kaikki -valinta vaikuttaa",
     "DESCRIPTION_CODE_FOLDING_MIN_FOLD_SIZE"         : "Rivejä vähintään ennen kuin pienennettävän osion kuvake tulee esiin",
     "DESCRIPTION_CODE_FOLDING_SAVE_FOLD_STATES"      : "true: muista pienennetyt osiot, jos tiedosto tai projekti suljetaan ja avataan sitten uudelleen",
     "DESCRIPTION_CODE_FOLDING_MAKE_SELECTIONS_FOLDABLE": "true: ota valitun tekstin koodin laskostus käyttöön editorissa",
-    "DESCRIPTION_ATTR_HINTS"                         : "Näytä tai piilota HTML-attribuuttien vihjeet",
+    "DESCRIPTION_ATTR_HINTS"                         : "Näytä tai piilota HTML-määritteiden vihjeet",
```
Comment (petetnt) [inline src/nls/fi/strings.js]: Would prefer the old one "attribuuttien"

**Review by petetnt — CHANGES_REQUESTED** [review]
LGTM with minor nitpicks

**Comment by valtlai** [comment]
Fixed

File: src/nls/fi/strings.js [hunk 5]
```diff
@@ -459,7 +459,7 @@ define({
     "PROJECT_SETTINGS_TITLE"               : "Projektin asetukset kohteelle: {0}",
     "PROJECT_SETTING_BASE_URL"             : "Esikatselun perus-URL",
     "PROJECT_SETTING_BASE_URL_HINT"        : "Käytä paikall. palvelinta antamalla URL, kuten http://localhost:8000/",
-    "BASEURL_ERROR_INVALID_PROTOCOL"       : "Esikatselu ei tue {0}-protokollaa. Käytä joko http: tai https: -mallia.",
+    "BASEURL_ERROR_INVALID_PROTOCOL"       : "Esikatselu ei tue {0}-protokollaa. Käytä joko http: tai https: \u2011muotoa.",
```
Comment (petetnt) [inline src/nls/fi/strings.js]: \u2011 was changed to `-` in other parts of this PR, maybe this should stay like so too? 🤔

**Review by petetnt — COMMENTED** [review]
One more small nit

File: src/nls/fi/strings.js (hunk 5 above)
Comment (valtlai) [inline src/nls/fi/strings.js]: Only in preferences descriptions, elsewhere it’s \u2011

**Review by petetnt — APPROVED** [review]

**Comment by petetnt** [comment]
Thanks once again!

**Pull request merged**